jarbas/core/models.py

```diff
@@ -124,8 +124,8 @@ def as_list(content, cast=None):
         if not content:
             return None
 
-        parts = list(content.split(','))
-        return list(map(lambda x: cast(x), parts)) if cast else parts
+        parts = content.split(',')
+        return list(map(cast, parts)) if cast else parts
```

Comment:
@cuducos it is up to you, my strategy was that of changing as little as possible